### DISCUSSION TITLE
ESP Home Issue while Upgrading to 2.3.6.

### DISCUSSION THREAD

**Discussion by adri123** [Blackymas/NSPanel_HA_Blueprint#1879] [General]
Hi, 
I encounter issue to update the 2.3.6 version of  NSPanel_HA_Blueprint .
The Blueprint and the TFT were updated without problem.
But I encounter issues with the firmwire update. When I lauch the update it freeze my HA server and then fail.

Version of my firmware : 2023.12.5, latest to update : 2024.2.2
Esphome version : the lastest up to date (I cannot tell right now as my HA is frozen).

Thank for your work and help.

**Comment by edwardtfn** [comment]
Please take a look at the ["Compiling Errors Guide"](https://github.com/Blackymas/NSPanel_HA_Blueprint/blob/dev/docs/error_compiling.md) and if the error persist please report as a bug in the [Issues area](https://github.com/Blackymas/NSPanel_HA_Blueprint/issues) so we can investigate it deeper.

**Reply by adri123** [reply]
Ok I tryed the guide, and its still not working : I think its a compiling error.
I will submite and issue then. 
Where can I have access to the compilator log ? I never see the error on the Esphome window (it just freeze).

**Reply by edwardtfn** [reply]
Could you please attach to the bug report a screenshot of that frozen screen so we can understand where is it failing?
And are you sure the system is really frozen? Couldn't be that it is still compiling in the background?

Well, please share in the bug report all the info you can and we can try to find something from that.

**Reply by adri123** [reply]
I allowed 4Gb of RAM instead of 2 GB to my VM and then it worked ! And now I'm on 4.3.0.
Maybe you should precise that in the ["Compiling Errors Guide"](https://github.com/Blackymas/NSPanel_HA_Blueprint/blob/dev/docs/error_compiling.md)

Thank you for make me feel haapy !

**Reply by edwardtfn** [reply]
This is there. 😉 

![image](https://github.com/Blackymas/NSPanel_HA_Blueprint/assets/94725493/87160083-7bd1-4635-9e08-86c0c434a2fa)

**Reply by adri123** [reply]
Damn !
